### PR TITLE
mgr/dashboard: Logs Page E2E Tests

### PR DESCRIPTION
**Pull request by adk3798**
Tests that logs page creates audit logs in response to pool creation/deletion
Tests that logs page creates audit logs in response to configuration changes
Alter configuration po to put config name in filter box so you can edit configs not on first page (first 10)

Fixes: https://tracker.ceph.com/issues/41035

Signed-off-by: Adam King <adking@redhat.com>
Signed-off-by: Rafael Quintero <rquinter@redhat.com>

### CODE REVIEW

**Comment by adk3798** [comment]
jenkins test dashboard

**Comment by adk3798** [comment]
jenkins test dashboard

File: src/pybind/mgr/dashboard/frontend/e2e/cluster/logs.po.ts [hunk 1]
```diff
@@ -1,8 +1,130 @@
...
+      .first()
+      .click();
+    $$('input.form-control.ng-valid')
+      .first()
+      .clear();
+    $$('input.form-control.ng-valid')
+      .first()
+      .sendKeys(configname);
+
+    const audit_logs_tab = element(by.cssContainingText('.tab-pane.active', ''));
```
Comment (tspmelo) [inline src/pybind/mgr/dashboard/frontend/e2e/cluster/logs.po.ts]:
```suggestion
    const audit_logs_tab = element(by.css('.tab-pane.active'));
```

File: src/pybind/mgr/dashboard/frontend/e2e/cluster/logs.po.ts [hunk 2]
```diff
@@ -1,8 +1,130 @@
...
+    const audit_logs_body = audit_logs_tab.element(by.cssContainingText('.card-body', ''));
```
Comment (tspmelo) [inline src/pybind/mgr/dashboard/frontend/e2e/cluster/logs.po.ts]:
```suggestion
    const audit_logs_body = audit_logs_tab.element(by.css('.card-body'));
```

File: src/pybind/mgr/dashboard/frontend/e2e/cluster/logs.e2e-spec.ts [hunk 3]
```diff
@@ -32,4 +45,49 @@ describe('Logs page', () => {
...
+      pools.exist(poolname, false);
+
+      logs.navigateTo();
+      logs.checkAuditForPoolFunction(poolname, 'delete', hour, minute);
+    });
+  });
+
+  describe('audit logs respond to editing configuration setting test', () => {
+    beforeAll(() => {
+      logs.navigateTo();
```
Comment (tspmelo) [inline src/pybind/mgr/dashboard/frontend/e2e/cluster/logs.e2e-spec.ts]:
```suggestion
```

File: src/pybind/mgr/dashboard/frontend/e2e/cluster/logs.po.ts [hunk 4]
```diff
@@ -1,8 +1,130 @@
...
+      .first()
+      .click();
+    $$('input.form-control.ng-valid')
+      .first()
+      .clear();
+    $$('input.form-control.ng-valid')
+      .first()
+      .sendKeys(poolname);
+
+    const audit_logs_tab = element(by.cssContainingText('.tab-pane.active', ''));
```
Comment (tspmelo) [inline src/pybind/mgr/dashboard/frontend/e2e/cluster/logs.po.ts]:
```suggestion
    const audit_logs_tab = element(by.css('.tab-pane.active'));
```

File: src/pybind/mgr/dashboard/frontend/e2e/cluster/logs.po.ts [hunk 5]
```diff
@@ -1,8 +1,130 @@
...
+    const audit_logs_body = audit_logs_tab.element(by.cssContainingText('.card-body', ''));
```
Comment (tspmelo) [inline src/pybind/mgr/dashboard/frontend/e2e/cluster/logs.po.ts]:
```suggestion
    const audit_logs_body = audit_logs_tab.element(by.css('.card-body'));
```

**Comment by adk3798** [comment]
Updates:
 - Changed some lines in logs po that used cssContainingText with no text to use css or just $
 - Changed lines getting filter box in configuration po to use $ instead of $$
 - Removed beforeAll block in new logs e2e Describe blocks since they were just calling navigateTo which is done within all It blocks within those Describe blocks

**Pull request merged**